EyeTrackApp/intensity_based_openness.py

```diff
@@ -213,7 +216,14 @@ def save(self):
     def change_roi(self, roiinfo: dict):
         self.now_roi[:] = [v for v in roiinfo.values()]
     
-    def intense(self, x, y, frame):
+    def clear_filter(self):
+        self.data = None
+        self.filterlist.clear()
+        self.averageList.clear()
+        if os.path.exists(self.imgfile):
+            os.remove(self.imgfile)
+
+    def intense(self, x, y, frame,filterSamples,outputSamples):
```

Comment:
So I've noticed this in a couple of places, it may be worth just straight up running Black over the file we're currently working as an additional step to format the code just to make it a bit cleaner 

https://github.com/psf/black   

we should also probably add it as a dev dependency too 
https://python-poetry.org/docs/managing-dependencies/

Reply:
I like that idea, formatting python doesn't seem very easy!